src/System.Runtime.Extensions/tests/System/Environment.Exit.cs

```diff
@@ -42,6 +42,15 @@ public static void CheckExitCode(int expectedExitCode)
 
         [Theory]
         [MemberData(nameof(ExitCodeValues))]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.UapNotUapAot, "RemoteExecutor behaves different in UAP.")]
```

Comment:
I got your point now. Sorry I didn't get that you meant that the test would become a one liner. I thought you meant to keep the using block and the tests as it was. I misunderstood completely. Thanks for the review.